onnxruntime/python/session.py

```diff
@@ -141,6 +141,12 @@ def end_profiling(self):
         """
         return self._sess.end_profiling()
 
+    def get_profiling_start_timeNs(self):
```

Comment:
```suggestion
    def get_profiling_start_time_ns(self):
```
(similar to https://docs.python.org/3/library/time.html#time.clock_gettime_ns)